src/package.json

```diff
@@ -18,8 +18,7 @@
     "lodash": "~4.17.11",
     "polyline-encoded": "~0.0.8",
     "riot-route": "~3.1.4",
-    "spinkit": "~1.2.5",
```

Comment:
Was spinkit intended to be removed? If so, there are still references to it in the code (eg, search for `sk-spinner`).

Reply:
No, it was not. Good catch!